dodgeBall/main.js

```diff
@@ -0,0 +1,159 @@
+const arrOfPeople = [
+    {
+        id: 2,
+        name: "Charles Young",
+        age: 55,
+        skillSet: "welding",
+        placeBorn: "Omaha, Nebraska"
+    },
+    {
+        id: 3,
+        name: "Judy Twilight",
+        age: 35,
+        skillSet: "fishing",
+        placeBorn: "Louisville, Kentucky"
+    },
+    {
+        id: 4,
+        name: "Cynthia Doolittle",
+        age: 20,
+        skillSet: "tic tac toe",
+        placeBorn: "Pawnee, Texas"
+    },
+    {
+        id: 5,
+        name: "John Willouby",
+        age: 28,
+        skillSet: "pipe fitting",
+        placeBorn: "New York, New York"
+    },
+    {
+        id: 6,
+        name: "Stan Honest",
+        age: 20,
+        skillSet: "boom-a-rang throwing",
+        placeBorn: "Perth, Australia"
+    },
+    {
+        id: 7,
+        name: "Mia Watu",
+        age: 17,
+        skillSet: "acrobatics",
+        placeBorn: "Los Angeles, California"
+    },
+    {
+        id: 8,
+        name: "Walter Cole",
+        age: 32,
+        skillSet: "jump rope",
+        placeBorn: "New Orleans, Louisiana"
+    },
+    ]
+
+    const listOfPlayers = [];
+    const blueTeam = [];
+    const redTeam = [];
+    //created playerc object with id, name, age, skillset, placeborn attributes
+    class PlayerC {
+    constructor(id, name, age, skillSet, placeBorn){
+        this.id = id,
+        this.name = name, 
+        this.age = age,
+        this.skillSet = skillSet,
+        this.placeBorn = placeBorn
+    }
+    }
+    //created an extension to playerc that adds a teamofcolor and mascot for the blue team 
+    class BlueTeammate extends PlayerC {
+        constructor(id, name, age, skillSet, placeBorn, teamOfColor, mascot){
+        super(id, name, age, skillSet, placeBorn)
+        this.teamOfColor = teamOfColor;
+        this.mascot = mascot;
+    }
+    }
+//created an extension to playerc that adds a teamofcolor and mascot for the red team 
```

Comment:
I would not have expected to see to classes, one for the BlueTeammate and one for the redTeammate
I was expecting to see a single class TeamMate, with an attribute that defines which team the teammate is in